### PR TITLE
[small]fix: fix build data

### PR DESCRIPTION
**Pull request by kumada626**
Fix buildData passed from api.
relates: screwdriver-cd/screwdriver#1361

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,13 +27,20 @@ const SCHEMA_SLACK_SETTINGS = Joi.object().keys({
...
         settings: SCHEMA_SLACK_SETTINGS.required(),
         status: SCHEMA_STATUS.required(),
-        pipelineName: Joi.string(),
+        pipeline: SCHEMA_PIPELINE_DATA.required(),
         jobName: Joi.string(),
-        buildId: Joi.number().integer(),
+        build: Joi.object().keys({
+            id: Joi.number().integer().required()
+        }).unknown(true),
+        event: Joi.object().required(),
```
Comment (tkyi) [inline index.js]: don't think `event` object is required for our notifications (at the moment)

**Comment by tkyi** [comment]
Could you also update `jenkins-mocha: ^6.0.0` in package.json?
https://github.com/screwdriver-cd/notifications-email/blob/master/package.json#L39

It's to resolve some vulnerabilities reported by npm install: https://cd.screwdriver.cd/pipelines/427/builds/74058

File: index.js (hunk 1 above)
Comment (kumada626) [reply index.js]: Remove required from `event` object.

**Comment by kumada626** [comment]
@tkyi Also update package.json to update `jenkins-mocha`.

**Pull request merged**